components/ChatApp.tsx

```diff
@@ -278,30 +278,38 @@ export class ChatApp extends React.Component<
                 channels: [...prevState.channels, channel.uniqueName],
             }));
         });
-        if (this.props.candidateName !== undefined) {
+        if (this.props.candidateName !== "") {
+            const channelName =
+                this.props.username + " - " + this.props.candidateName;
             // Si un candidate a été invité
             const previousChannel = this.channel || undefined;
-            try {
+            // false === channel non existant, true === channel déjà crée
+            let created = false;
+            const paginator = await this.chatClient.getSubscribedChannels(
+                undefined,
+            );
+            created = paginator.items.some(
+                (value: any, index: number, array: any) => {
```

Comment:
rename value to channel, remove unused args, but before, config tslint to display warning on unused params